src/app/surveys/surveys.component.ts

```diff
@@ -84,9 +84,10 @@ export class SurveysComponent implements OnInit, AfterViewInit, OnDestroy {
 
   openSendSurveyDialog(survey) {
     forkJoin([
-      this.dialogsListService.getListAndColumns('_users'),
```

Comment:
Rather than changing this here, we should change the behavior in `DialogsListService`